.gitignore

```diff
@@ -30,7 +30,7 @@ config/services.yml
 
 ###> squizlabs/php_codesniffer ###
 /.phpcs-cache
-/phpcs.xml
+/ci/qa/phpcs.xml
```

Comment:
Why do you want to git-ignore the phpcs config file? Don't be fooled by the boilerplate code squizlabs put in our `.gitignore`